### PR TITLE
Issue #179 Import Older magazine issues

### LINKED ISSUE #179: Import Older Magazines
https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B5BB10087-2C46-4DED-A9E2-A42BD882AC6E%7D&file=import-status.xlsx&action=default&mobileredirect=true

### PR DESCRIPTION
**Pull request by iamitshri**
Fix #179

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/
- After:  
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-1-issue-2
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-1-issue-1
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-1-issue-4
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-2
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-1-issue-3
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-1
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-4
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-1-issue-5
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-5
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-6
- https://179-import-older-magazines--24life--hlxsites.hlx.page/drafts/ashrigondeka/old-magazines/volume-2-issue-3

### CODE REVIEW

File: tools/importer/import_older_magazines.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+}
+
+/*
+ * content structure: in first cell:
+ *  - p > image
+ *  - p > COLLECTIONS (optional)
+ *  - title as linked h1, h2, h3, h4, h5, or h6
+ *  - p > author with link (optional)
+ */
+function createCardBlocks(main, document) {
```
Comment (iamitshri) [inline tools/importer/import_older_magazines.js:115]: This wasn't changed from the original magazine import

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -58,6 +58,104 @@ div.column-collections-parent {
...
+}
+
+.columns.old-magazine-header.block .header-content{
+  animation: slide-in-from-right 1s ease-out forwards;
+  transform:translateX(-300%);
+  opacity: 0;
+}
+
+
+@keyframes slide-in-from-right {
```
Comment (synox) [inline blocks/columns/columns.css:72]: looks great. We don't need the animation, we agreed with the customer that it's a better experience without it

Comment (iamitshri) [reply blocks/columns/columns.css:72]: sure I will take it out in a min

**Review by synox — APPROVED** [review]
great! Thank you

**Pull request merged**